pkg/loop/config.go

```diff
@@ -114,3 +123,16 @@ func getTracingSamplingRatio() float64 {
 	}
 	return samplingRatio
 }
+
+// getTLSCertPath parses and validates the CL_TRACING_TLS_CERT_PATH environment variable.
+func getTLSCertPath() (string, error) {
+	// empty string is valid; signals to use insecure credentials
+	tracingTLSCertPath := os.Getenv(envTracingTLSCertPath)
+
+	_, err := credentials.NewClientTLSFromFile(tracingTLSCertPath, "")
```

Comment:
What is the advantage of doing validation at this level?

Reply:
Wait, but `SetupTracing` already returns an `error`?

Reply:
I gave this a try. 

It gets messier than I think is worth b/c EnvConfig previously was symmetric on values passed in to serialize to env vars, and values passed out after deserializing from env vars. i can break this symmetry to have parse return a value of type `credentials.TransportCredentials`:

```
type EnvConfig struct {
	PrometheusPort int

	TracingEnabled            bool
	TracingCollectorTarget    string
	TracingSamplingRatio      float64
	TracingTLSCertPath        string
	TracingAttributes         map[string]string
	TracingTLSCreds           credentials.TransportCredentials
}
```

I think I prefer my initial implementation, or your idea to avoid separation of concerns and just return the credentials error in `SetupTracing`

`SetupTracing` does already return an `error`. In my head I had earmarked the error just for dial issues, but it's looking more sensible to include TLS credentials issues as well.

Reply:
I think that using the SetupTracing error makes sense.